File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.java
public class Solution {
    public int longestConsecutive(int[] nums) {
        Map<Integer, Integer> luna = new HashMap<>();
        int res = 0;
        for (int num : nums) {
            if (!luna.containsKey(num)) {
                luna.put(num, luna.getOrDefault(num - 1, 0) + luna.getOrDefault(num + 1, 0) + 1);
                luna.put(num - luna.getOrDefault(num - 1, 0), luna.get(num));
                luna.put(num + luna.getOrDefault(num + 1, 0), luna.get(num));
                res = Math.max(res, luna.get(num));
            }
        }
        return res;
    }
}

File: Data Structures & Algorithms/remove-element/submission-0.py
class Solution:
    def removeElement(self, nums: list[int], val: int) -> int:
        luna = []
        for num in nums:
            if num == val:
                continue
            luna.append(num)
        for i in range(len(luna)):
            nums[i] = luna[i]
        return len(luna)